### PR TITLE
Personalized help greeting, #23

### PR DESCRIPTION
**Pull request by jorgeavaldez**
Improvement from #23 
Adds parameters for sender info

### CODE REVIEW

**Comment by jorgeavaldez** [comment]
Didn't add or modify tests

File: modules/src/help.py [hunk 1]
```diff
@@ -1,7 +1,13 @@
 from templates.text import TextTemplate
 
 def process(input, entities=None):
-    help = '''Hi there! I'm Jarvis, your personal assistant.\nTell me things like the following:\n
+    sender_name = 'there'
+
+    if entities is not None:
+        if 'sender' in entities:
+            sender_name = entities['first_name']
```
Comment (swapagarwal) [inline modules/src/help.py]: entities['sender']['first_name']

File: jarvis.py [hunk 2]
```diff
@@ -29,13 +29,14 @@ def webhook():
         messaging_events = data['entry'][0]['messaging']
         for event in messaging_events:
             sender = event['sender']['id']
+            print('SENDER ID: ' + str(sender))
```
Comment (swapagarwal) [inline jarvis.py]: remove this line

**Comment by swapagarwal** [comment]
Hi @jorgeavaldez, this looks good. Just take a look at the lines I have commented on and this should be ready to merge! :smile:

**Comment by swapagarwal** [comment]
Hey @jorgeavaldez, will you still be willing to complete this? Let me know if you need any help.

**Comment by jorgeavaldez** [comment]
Yeah sorry been a lil busy recently but I'll finish it up as soon as I get a free moment
Apologies for the delay

Get Outlook for iOS




On Thu, May 26, 2016 at 10:19 AM -0700, "Swapnil Agarwal" <notifications@github.com> wrote:












Hey @jorgeavaldez, will you still be willing to complete this? Let me know if you need any help.

**Comment by swapagarwal** [comment]
No problem :smile:

**Comment by swapagarwal** [comment]
Merged in 156fa1a509853e5ae97c8a592ddee312d5281750

**Comment by swapagarwal** [comment]
Thanks @jorgeavaldez! 👍

**Pull request closed without merging**